homeassistant/components/media_player/squeezebox.py

```diff
@@ -203,9 +188,28 @@ def state(self):
                 return STATE_IDLE
         return STATE_UNKNOWN
 
-    def update(self):
-        """Retrieve latest state."""
-        self._status = self._lms.get_player_status(self._id)
+    def async_query(self, *parameters):
+        """Send a command to the LMS."""
```

Comment:
Last request here: please update this docstring to include the second line based on the convention in #5001.